internal/domain/bonustransactions/pg.go

```diff
@@ -0,0 +1,212 @@
+package bonustransactions
+
+import (
+	"context"
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type Repo struct {
+	Con *pgxpool.Pool
+}
+
+func NewRepo(con *pgxpool.Pool) *Repo {
+	return &Repo{
+		con,
+	}
+}
+
+func (r *Repo) List(ctx context.Context, pars *ListPars) ([]*BonusTransaction, error) {
+	queryBuilder := squirrel.Select("*").From("bonus_transactions").Where(squirrel.Eq{"true": true})
+	var values = make(map[string]interface{})
+
+	if pars.ProcessedAfter != nil && !pars.ProcessedAfter.IsZero() {
+		values["processed_at"] = pars.ProcessedAfter
+		queryBuilder = queryBuilder.Where(squirrel.GtOrEq{"processed_at": values["processed_at"]})
```

Comment:
```suggestion
		queryBuilder = queryBuilder.Where(squirrel.GtOrEq{"processed_at": pars.ProcessedAfter})
```
а зачем в мапу складывать? Можно сразу значение передать